app/controllers/katello/api/v2/capsule_content_controller.rb

```diff
@@ -51,7 +51,7 @@ def remove_lifecycle_environment
     param :environment_id, Integer, :desc => N_('Id of the environment to limit the synchronization on')
     def sync
       find_environment if params[:environment_id]
-      task = async_task(::Actions::Katello::CapsuleContent::Sync, capsule_content, :environment => @environment)
+      task = async_task(::Actions::Katello::CapsuleContent::Sync, capsule_content.capsule, :environment_id => @environment.try(:id))
```

Comment:
With BulkAction, attempting to pass a hash containing AR resources resulted in plan phase errors similar to below; however, passing in a hash of IDs worked fine.
```
undefined method `to_hash' for #<Katello::ContentView:0x007f70248aa480>
```

I haven't found an alternative approach for passing in the resources, aside from passing in an array of arguments vs a hash.  If you have suggestions, please do let me know.  In most of Katello's usage of BulkAction, it does pass an array of arguments; however, I leaned away from it here, since we invoke the action differently depending upon scenario (e.g. cv + env vs repo).